src/playground/App.js

```diff
@@ -21,6 +21,28 @@ const rulesMeta = Array.from(rules.entries()).reduce((result, [key, value]) => {
     return result;
 }, {});
 
+const fillOptionsDefaults = (options = {}) => ({
+    ...options,
+    env: {
+        es6: true,
```

Comment:
We should add `es6` only if there were no `urlOptions || storedOptions`. Otherwise, if someone removes it and opens the playground again `es6` would mistakenly reappear.